themes/bootstrap3/templates/RecordDriver/DefaultRecord/list-entry.phtml

```diff
@@ -33,36 +33,37 @@
     <?php endif; ?>
     <div class="media-body">
       <div class="result-body">
-        <div class="resultItemLine1">
-          <?php $missing = $this->driver instanceof \VuFind\RecordDriver\Missing; ?>
-          <?php if (!$missing): ?><a href="<?=$this->escapeHtmlAttr($this->recordLinker()->getUrl($this->driver))?>" class="getFull" data-view="<?=$this->params->getOptions()->getListViewOption() ?>"><?php endif; ?>
-            <span class="title"><?=$this->record($this->driver)->getTitleHtml()?></span>
-          <?php if (!$missing): ?></a><?php endif; ?>
-          <?php foreach ($this->driver->tryMethod('getTitlesAltScript', [], []) as $altTitle): ?>
-            <div class="title-alt">
-              <?=$this->escapeHtml($altTitle)?>
-            </div>
-          <?php endforeach; ?>
-        </div>
+        <header>
+          <h2>
+            <?php $missing = $this->driver instanceof \VuFind\RecordDriver\Missing; ?>
+            <?php if (!$missing): ?><a href="<?=$this->escapeHtmlAttr($this->recordLinker()->getUrl($this->driver))?>" class="getFull" data-view="<?=$this->params->getOptions()->getListViewOption() ?>"><?php endif; ?>
+              <span class="title"><?=$this->record($this->driver)->getTitleHtml()?></span>
+            <?php if (!$missing): ?></a><?php endif; ?>
+            <?php foreach ($this->driver->tryMethod('getTitlesAltScript', [], []) as $altTitle): ?>
+              <div class="title-alt">
+                <?=$this->escapeHtml($altTitle)?>
+              </div>
+            <?php endforeach; ?>
+          </h2>
 
-        <div class="resultItemLine2">
           <?php if ($this->driver->isCollection()): ?>
-            <?=implode('<br>', array_map([$this, 'escapeHtml'], $this->driver->getSummary())); ?>
+            </header>
+            <div class="resultItemLine1">
+              <?=implode('<br>', array_map([$this, 'escapeHtml'], $this->driver->getSummary())); ?>
           <?php else: ?>
             <?php $summAuthors = $this->driver->getPrimaryAuthors(); if (!empty($summAuthors)): ?>
               <?=$this->transEsc('by')?>
               <?php $authorCount = count($summAuthors); foreach ($summAuthors as $i => $summAuthor): ?>
                 <a href="<?=$this->record($this->driver)->getLink('author', $summAuthor)?>"><?=$this->escapeHtml($summAuthor)?></a><?=($i + 1 < $authorCount ? ';' : '') ?>
               <?php endforeach; ?>
             <?php endif; ?>
-
+            </header>
+            <div class="resultItemLine1">
             <?php $journalTitle = $this->driver->getContainerTitle(); $summDate = $this->driver->getPublicationDates(); ?>
             <?php if (!empty($journalTitle)): ?>
-              <?=!empty($summAuthor) ? '<br>' : ''?>
```

Comment:
I notice that some line breaks are getting removed here (and in some other places as well). Is this intentional? Have its possible consequences been tested/inspected?

Reply:
Yes, that was intentional, because a line break is already done by the header tag. Keeping the breaks would create empty lines.